styles.css

```diff
@@ -589,4 +688,53 @@ hr {
   .contact-btn {
     margin: 5px 30%;
   }
+
+  .popup-content {
+    width: 1156px;
+    height: 570px;
+    border-radius: 24px;
+    display: block;
+    gap: 20px;
+    padding: 0 auto;
+    margin: 20px auto;
+    overflow-y: scroll;
+  }
```

Comment:
- [ ] I have noticed that your `popup-content` has a fixed width. This breaks the responsiveness of the popup in different screen sizes. Kindly consider using the percentage width to match the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=u9NC5n6sAS6bpo65-0) 😎
- [ ] In addition, kindly ensure that the height of the popup content matches the Figma design. You may need to consider reducing the height accordingly ☺️.

Your design at 900px
![image](https://github.com/manzitresor/Portfolio/assets/99394374/0a2264c9-33be-4083-a456-5db84276255d)

Figma design 😎 
![image](https://github.com/manzitresor/Portfolio/assets/99394374/df011c02-6672-43bf-9791-8e0a53c99e9c)